guides/user/kubernetes-v2/deploy-helm/index.md

```diff
@@ -25,7 +25,8 @@ When configuring the "Bake (Manifest)" stage, you can specify the following:
 * __The release name__ (required)
 
   The Helm release name for this chart. This determines the name of the
-  artifact produced by this stage.
+  artifact produced by this stage. **Note: this name will override any
+  changes you make to the name in the Produces Artifacts section.**
```

Comment:
nit: consider formatting this the way we format other notes in this file (i.e., indenting via "> " rather than bolding it).

Reply:
👍 fixed